vendor/priors/gwt.js

```diff
@@ -1288,11 +1320,24 @@ function directGetEntryDataFromCacheProm(key) {
     });
     return future;
 }
-
+function decodeEntryData(val, future) {
+    let binary = window.atob(val);
+    var data = new Int8Array(binary.length);
+    for (var i = 0; i < binary.length; i++) {
+        data[i] = binary.charCodeAt(i);
+    }
+    future.complete(convertToByteArray(data));
+}
 function clearAccountCacheFully(func) {
     if (accountStoreCache.isCachingEnabled) {
         clearIDBKV(accountStoreCache.cacheAccountStore).then(res => func());
     } else {
+        for (var i = 0; i < window.localStorage.length; i++) {
+            let key = window.localStorage.key(i);
+            if (key != 'rootKey') {
```

Comment:
Would it be safer to have a prefix for the key here? In case we use localstorage for something else?